tutorials/introductory/getting_started.py

```diff
@@ -0,0 +1,767 @@
+#
+# The image below depicts each visible element of a Matplotlib graph. The
+# graphic uses Matplotlib to display and highlight each individual part of the
+# visualization. To see how the programming operates, the source code is
+# available at :ref:`sphx_glr_gallery_showcase_anatomy.py`.
+#
+# .. note::
+#
+#     ``Figure`` and ``Axes`` identify empty regions of the diagram;
+#     however, these elements are foundational in operation.
+#
+# .. image:: ../../_static/anatomy.png
+#
+# :class:`~matplotlib.figure.Figure`
+#
+# The Figure is the working space for the programming. All visible
+# objects on a graph are located within the Figure.
+#
+# :class:`~matplotlib.axes.Axes`
+#
+# Axes are subplots within the Figure. They contain Figure elements and
+# are responsible for plotting and configuring additional details. Each
+# Figure can contain multiple Axes, but each Axes is specific to one
+# Figure.
+#
+# In a Figure, each Axes can contain any number of visual elements. An Axes may
+# have more than one type of visualization of data. From the `Plotting`_
+# section above, the Axes in both explicit and implicit strategies contain
+# multiple types of visualizations of data on a single Axes. Each of these
+# types are specific to the Axes they are in.
+#
+# Other Components
+# ^^^^^^^^^^^^^^^^
+#
+# :class:`~matplotlib.artist.Artist`
+#
+# Artists are broad Matplotlib objects that display visuals. These are the
+# visible elements when the Figure is rendered. They correspond to a specific
+# Axes and cannot be shared or transferred. In Matplotlib programming, all
+# objects for display are Artists.
+#
+# .. note::
+#
+#   Axes and Axis are not synonymous. Axis refers to
+#   :class:`~matplotlib.axis.Axis`.
+#
+# Manipulating Artists
+# --------------------
+#
+# With simple plots, Matplotlib automatically generates the basic elements of
+# a graph. For more control over the visual, use Artists and its methods.
+#
+# Matplotlib generates additional visual elements as Artists in the form of
+# objects. As Artists, each has its own respective methods and functions.
+# Explicit and implicit approaches use different methods and are not
+# interchangeable.
+#
+# The table below compares common Artists and their different methods.
+#
+# +-----------------------+--------------------------+------------------------+
+# | Artist                | Explicit                 | Implicit               |
+# +=======================+==========================+========================+
+# | Visible elements from | Each specific Axes has   | The ``pyplot`` module  |
+# | rendered Figure.      | its own method for       | manages Artists based  |
+# |                       | artists.                 | on most recent Figure  |
+# |                       |                          | or Axes.               |
+# +-----------------------+--------------------------+------------------------+
+# | X-axis labels         | ``ax.set_xticks()``      | ``plt.xlabel()``       |
```

Comment:
would maybe combine the x/y ticks and then link out to locators, seperate for x/y ticklabels w/ link out to formatters, and ax.set_xticks/xticklabels = plt.xticks(ticks, labels=labels), ax.set_xlabel=plt.xlabel